internal/proxy/task_query.go

```diff
@@ -361,16 +361,7 @@ func (t *queryTask) PreExecute(ctx context.Context) error {
 		t.RetrieveRequest.Username = username
 	}
 
-	if t.request.TravelTimestamp == 0 {
-		t.TravelTimestamp = t.BeginTs()
-	} else {
-		t.TravelTimestamp = t.request.TravelTimestamp
-	}
-
-	err = validateTravelTimestamp(t.TravelTimestamp, t.BeginTs())
-	if err != nil {
-		return err
-	}
+	t.TravelTimestamp = t.BeginTs()
```

Comment:
why to we change the segcore to remove t.TravelTimestamp?

Reply:
but why use t.BeginTs as travel timestamp?
what's the difference between beginTs and endTs

Reply:
For task_query, beginTs and endTs are the same, both are t.Base.Timestamp, using the time of task enqueue.

In this PR, there are no changes to the settings. Only the "traveltimestamp" in the request has been removed,  is the default value is used as it was before when "traveltimestamp" was not set.